vision/cloud-client/src/test/java/com/example/vision/DetectIT.java

```diff
@@ -124,7 +124,7 @@ public void testLandmarksGcs() throws Exception {
   @Test
   public void testLandmarksUrl() throws Exception {
     // Act
-    String[] args = {"landmarks", "https://pbs.twimg.com/media/C4wfMOhVcAAUAgq.jpg"};
+    String[] args = {"landmarks", "http://wheresgus.com/landmark.jpg"};
```

Comment:
I'm not sure if we have a public bucket. In Ruby, vision sample tests upload the image before testing a sample. 

I do agree, It may be a good idea to have a public bucket we can all use in our samples.